packages/contracts-versions/create-contract-versions.ts

```diff
@@ -22,8 +32,17 @@ async function buildContracts(commit: string) {
   }
 }
 
-async function copyActiveContracts(commit: string, versionName: string) {
+async function copyContracts(versionName: string) {
   try {
+    console.log(`Copying contracts source code`);
```

Comment:
I would print `Copying contracts/src to ${destPath}`

Reply:
amended.